integrationtests/src/test/java/org/neo4j/procedure/ProcedureIT.java

```diff
@@ -1278,12 +1292,12 @@ public NodeListRecord( List<Node> nodes )
         public TerminationGuard guard;
 
         @Context
-        public KernelTransaction ktx;
```

Comment:
Now we are no longer using any unsafe component here... should we reflect that somehow in the test? It feels nice in general!